samples/SourceUpdaterSample/SourceUpdaterSample/MakeConstCodeFixProvider.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Formatting;
+
+namespace SourceUpdaterSample
+{
+    /// <summary>
+    /// As with the analzyers, code fix providers that are registered into Upgrade Assistant's
+    /// dependency injection container (by IExtensionServiceProvider) will be used during
+    /// the source update step.
+    /// </summary>
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = "UA001 CodeFix Provider")]
```

Comment:
Doesn't look like the name `UA001` overlaps with any of our existing providers but I think a different Name sample would help users see the difference between their extensions and our analyzers.

Could save some time during the debugging process some day.

Reply:
Good point. I've changed this to 'Sample1 CodeFix Provider'